python/res/enkf/jobs/scaling/measured_data.py

```diff
@@ -202,30 +223,32 @@ def _combine_filters(filters):
         return combined_filter
 
     @staticmethod
-    def _add_summary_data(ert, data_key, observation_key, case_name):
+    def _add_summary_data(ert, data_key, case_name):
         data = SummaryCollector.loadAllSummaryData(ert, case_name, [data_key])
         data = data[data_key].unstack(level=-1)
-        return data.set_index([[observation_key] * len(data)])
+        return data.set_index(data.index.values)
 
     @staticmethod
-    def _add_summary_observations(ert, data_key, observation_key, case_name):
+    def _add_summary_observations(ert, data_key, case_name):
         data = SummaryObservationCollector.loadObservationData(
             ert, case_name, [data_key]
         ).transpose()
         data = data.set_index(
```

Comment:
I think an explanation would help here. Why are you doing this?

Reply:
The restructure is from:
index                  |   values  ...   
OBS_{KEY}       | ...
STD_{KEY}       | ....
{KEY}                | ...
{KEY}                | ...
OBS_{KEY_2}   | ...
STD_{KEY_2}   | ....
{KEY_2}            | ...
{KEY_2}            | ...

to:
{KEY}                                            {KEY_2}
----------------------------------------------------------------------------------
index        |          values
OBS         |                                   
STD         |
{REAL_1}   |
{REAL_2}   |